docs/source/API.rst

```diff
@@ -12,4 +12,7 @@ API for pycgm
    Pipelines
    pyCGM_Helpers
    pycgmCalc
-   pycgmStatic
\ No newline at end of file
+   pycgmStatic
+   clusterCalc
+   runpyCGM
+   pycgm_embed
```

Comment:
pycgm_embed shouldn't be under API. I think we could move it to a 'getting started' type section though.